### PR TITLE
Added links to the pos page

### PR DESCRIPTION
**Pull request by calvinbrown085**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -165,5 +165,12 @@ def createManagerSale():
     newSalePrice = request.args.get('newPrice')
     createSale(productId, newSalePrice)
     return redirect("/managerPage")
+
+
+@app.route("/cartClear")
+def clearCart():
+    session["resultList"] = []
+    session["totalAmount"] = 0.00
```
Comment (ethanphunter) [inline main.py:173]: Does this work? In the user login package I have it being set to `0`. I'm not sure if it makes a difference though.

**Comment by ethanphunter** [comment]
Other than my question :+1:

File: main.py (hunk 1 above)
Comment (calvinbrown085) [inline main.py:173]: I tested it and it didn't seem to care @ethanphunter

Comment (ethanphunter) [inline main.py:173]: Awesome

**Comment by ethanphunter** [comment]
👍  👍  👍

**Comment by calvinbrown085** [comment]
plz merge? @ethanphunter

**Comment by ethanphunter** [comment]
Fine..... I guess we can get rid of the 2 approvals since it's so late....

**Pull request merged**